arithmetization/src/main/java/net/consensys/linea/plugins/rpc/capture/CaptureEndpointServicePlugin.java

```diff
@@ -63,6 +65,11 @@ private void createAndRegister(
         method.getNamespace(), method.getName(), method::execute);
   }
 
+  @Override
+  public Map<String, LineaOptionsPluginConfiguration> getLineaPluginConfigMap() {
+    return Map.of();
```

Comment:
Maybe here doesn't make a difference, but in general should call `var superMap = super.getLineaPluginConfigMap()` and add local mapping to the `superMap`